src/mina-signer/mina-signer.ts

```diff
@@ -39,17 +39,10 @@ export { Client, Client as default, type NetworkId };
 const defaultValidUntil = '4294967295';
 
 class Client {
-  private network: NetworkId; // TODO: Rename to "networkId" for consistency with remaining codebase.
+  private network: NetworkId;
 
-  constructor(options: { network: NetworkId }) {
-    if (!options?.network) {
-      throw Error('Invalid Specified Network');
-    }
-    const specifiedNetwork = options.network.toLowerCase();
```

Comment:
this is no longer needed since the type now is `'testnet' | 'mainnet' ` which already is lower case